pkg/logging/http.go

```diff
@@ -72,12 +76,14 @@ func HTTPMiddleware(logger *zap.Logger, dump bool, handler http.Handler) http.Ha
 			}
 		}
 
-		subject, ok := r.Context().Value(CtxAuthSubject{}).(string)
-		if !ok {
-			authErrorMsg := r.Context().Value(CtxAuthError{}).(string)
-			logger = logger.With(zap.String("resp_sub_err", authErrorMsg))
-		} else {
-			logger = logger.With(zap.String("req_sub", subject))
+		if errMsgValue := r.Context().Value(CtxKeyErrMsg); errMsgValue != nil {
+			if errMsg, ok := errMsgValue.(string); ok && errMsg != "" {
+				logger = logger.With(zap.String("resp_sub_err", errMsg))
+			}
```

Comment:
I do not think it is really necessary within the logging middleware to get the JWT subject decoding error, if any. In case of error let's just not put anything in the context. The error will be handled by the authorizer anyway. And propagating an error within the context is already not a very much common practice, so let's not do it twice. 

So I suggest to remove this and `CtxKeyErrMsg`.